scripts/publish-ghpages.sh

```diff
@@ -0,0 +1,27 @@
+#/bin/sh
+
+set -o errexit -o nounset
+
+# User info
+git config user.name "patternfly-build"
+git config user.email "patternfly-build@redhat.com"
+git config --global push.default simple
+
+## Deploy to github pages
+if [ -d "patternfly.github.io" ]; then
+  rm -rf patternfly.github.io
+fi
+git clone git@github.com:patternfly/patternfly.github.io.git
+rsync -av --delete --exclude .git source/_site/ patternfly.github.io
+cd patternfly.github.io
```

Comment:
Curious about the need for a separate patternfly.github.io repo. Is that a GitHub Pages requirement? Can we not use another branch (e.g., master-dist) in patternfly/patternfly.org?

My concern is that if patternfly, angular-patternfly, and RCUE move to GitHub, will we need to create 4 additional repos?